docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -2,6 +2,21 @@
 Node scheduler properties
 =========================
 
+Configuration
+-------------
+
+``node-scheduler.include-coordinator``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``false``
+
+Allows scheduling work on the coordinator so that a single machine can function
+as both coordinator and worker. For large clusters, processing work on the
+coordinator can impact query performance because the machine's resources are not
```

Comment:
Should we say "can negatively impact" or "can worsen" instead of just "can impact" to be a little more explicit about it?

Reply:
Good catch.